src/crimson/os/seastore/cached_extent.h

```diff
@@ -143,7 +233,9 @@ class CachedExtent : public boost::intrusive_ref_counter<
    * structure which defers updating the actual buffer until
    * on_delta_write().
    */
-  virtual CachedExtentRef duplicate_for_write() = 0;
+  CachedExtentRef duplicate_for_write(Transaction &t);
+
+  virtual CachedExtentRef get_mutable_replica(Transaction &t) = 0;
```

Comment:
Same here, all duplicate for write does is set invoke get_mutable_replica and then update touched_by.  Let's just call it duplicate_for_write and duplicate_for_write_impl.

Reply:
The duplicate_for_write comment above needs to be fixed since duplicate_for_write doesn't get implemented directly and the new virtual method also needs a comment.  Same for on_initial_write and duplicate_for_write.

Reply:
ok